package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@la-ots/pelican",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
You might need to merge main into this branch as this is already the current working version. Additionally we typically don't bump the version number if there isn't a change to the design system itself.

Reply:
Not in the lock file it is.